integration_tests/pages/imprisonmentStatus.ts

```diff
@@ -0,0 +1,26 @@
+import Page, { PageElement } from './page'
+
+export default class ImprisonmentStatusPage extends Page {
+  constructor() {
+    super('What is the reason for imprisonment?')
+  }
+
+  static goTo(id: string): ImprisonmentStatusPage {
+    cy.visit(`/prisoners/${id}/imprisonment-status`)
+    return Page.verifyOnPage(ImprisonmentStatusPage)
+  }
+
+  errorSummaryTitle = (): PageElement => cy.get('#error-summary-title')
+
+  errorSummaryBody = (): PageElement => cy.get('.govuk-error-summary__body')
+
+  errorSummaryMessage = (): PageElement => cy.get('.govuk-error-message')
+
+  prisonerName = (): PageElement => cy.get(`[data-qa=prisoner-name]`)
+
+  imprisonmentStatusSingleReasonRadioButton = (): PageElement => cy.get(`[data-qa=imprisonment-status-1]`)
+
+  imprisonmentStatusMultipleReasonRadioButton = (): PageElement => cy.get(`[data-qa=imprisonment-status-3]`)
```

Comment:
It's clearer to use value instead rather than tying it to specific indexes: 

https://github.com/ministryofjustice/use-of-force/blob/e649c0618efc4d32e5e1c8141241264c9e6c3aca/integration-tests/pages/createReport/selectPrimaryUofReasonPage.js#L8

https://github.com/ministryofjustice/use-of-force/blob/main/integration-tests/integration/createIncident/form-submit.spec.js#L76